pom.xml

```diff
@@ -1824,9 +1824,7 @@
                 <exclude>**/RegexpHeaderCheckTest.class</exclude>
 
                 <!-- all bellow until https://github.com/checkstyle/checkstyle/issues/11446 -->
-                <exclude>**/AbstractFileSetCheckTest.class</exclude>
```

Comment:
I was going through this issue related PRs and came across this one, I just wanted to know the reason behind removing this test from the list given that you are working on Treewalker test only.

Reply:
@suniti0804 Actually I worked on this test AbstractFileSetCheckTest and didn't removed this line unknowingly. So I thought to remove the confusion for other contributions , lets remove this line in this PR only. Thats why i removed this line !

Reply:
please make this line update in separate PR, to merge it intstantly.